plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemRawNBT.java

```diff
@@ -86,6 +86,8 @@ public ItemRawNBT(ItemTag _item) {
                 "Items",
                 // Goat Horn specific
                 "instrument",
+                // Armor specific
+                "trim",
```

Comment:
Very nitpicky, but trims aren't actually armor specific as you can make them apply to any item with data/resource packs iirc